packages/i18n/src/Controller.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState, useEffect, useContext, createContext} from 'react';
+import i18n from 'i18n-js';
+
+import locales from '../locales';
+
+interface I18nContextProps {
+  t: (key: string, options: object) => string;
+  setLocale: (locale: string) => void;
+  locale: string;
+}
+
+const I18nContext = createContext<I18nContextProps | undefined>(undefined);
+
+const I18nProvider: React.FC = ({children}) => {
+  const [locale, setLocale] = useState<string>('pt-BR');
+
+  i18n.fallbacks = true;
+  i18n.defaultLocale = 'pt-BR';
+
+  useEffect(() => {
+    i18n.locale = locale;
```

Comment:
Shouldn't we persist this data to handle better communications/email? Like, this should be a user preference.

Reply:
So what do you recommend that we do?

Reply:
Maybe create a screen accessible by configuration menu where the user can switch the app language and persist this info on the backend side. What do you think about doing this in another task/pr @renanmav?